src/pages/components/sheet.mdx

```diff
@@ -0,0 +1,200 @@
+
+<CodeExample code={SheetHeadersCode}>
+  <SheetHeaders />
+</CodeExample>
+
+### Placement
+
+Sheets can be placed at the start or end of the container. End is the default placement.
+
+import SheetPlacement from '../../examples/chrome/SheetPlacement.tsx';
+import SheetPlacementCode from '!!raw-loader!../../examples/chrome/SheetPlacement.tsx';
+
+<CodeExample code={SheetPlacementCode}>
+  <SheetPlacement />
+</CodeExample>
+
+### Size
+
+Sheets default to 380px in width. You can customise the width based on your intended
+experience to any size using pixels or our [grid system](/components/grid).
+
+import SheetSize from '../../examples/chrome/SheetSize.tsx';
+import SheetSizeCode from '!!raw-loader!../../examples/chrome/SheetSize.tsx';
+
+<CodeExample code={SheetSizeCode}>
+  <SheetSize />
+</CodeExample>
+
+## API
+
+The Sheet component follows this structure:
+
+<!-- prettier-ignore -->
+```jsx
+<Sheet>
+  <Sheet.Header>
+    <Sheet.Title>
+      {/* a text to label the sheet */}
+    </Sheet.Title>
+    <Sheet.Description>
+      {/* a text describing the sheet */}
+    </Sheet.Description>
+  </Sheet.Header>
+  
+  <Sheet.Body>
+    {/* main content of the sheet */}
+  </Sheet.Body>
+
+  <Sheet.Footer>
+    <Sheet.FooterItem>
+      {/* items to set in the footer, such as buttons */}
+    </Sheet.FooterItem>
+  </Sheet.Footer>
+
+  <Sheet.Close />
+</Sheet>
+```
+
+Please keep the subcomponent structure in the order displayed in the example above.
+
+### Sheet
+
+<Component components={props.data.mdx.components} componentName="sheet" />
+
+The Sheet component provides state and accessibility attributes to its children.
+
+<PropSheet components={props.data.mdx.components} componentName="sheet" />
+
+#### Sheet.Body
+
+<Component components={props.data.mdx.subcomponents} componentName="sheet.body" />
+
+A Body requires additional accessible functionality to make the
+element focusable for keyboard scrolling. Nest it within the [Sheet](#sheet) component.
+
+<PropSheet components={props.data.mdx.subcomponents} componentName="sheet.body" />
+
+#### Sheet.Title
+
+<Component components={props.data.mdx.subcomponents} componentName="sheet.title" />
+
+A Title labels the Sheet for assistive technology. Nest it within a [Header](#sheetheader) component.
+
+<PropSheet components={props.data.mdx.subcomponents} componentName="sheet.title" />
+
+#### Sheet.Description
+
+<Component components={props.data.mdx.subcomponents} componentName="sheet.description" />
+
+A Description describes the Sheet for assistive technology.
+Nest it within a [Header](#sheetheader) component.
+
+<PropSheet components={props.data.mdx.subcomponents} componentName="sheet.description" />
+
+#### Sheet.Header
+
+<Component components={props.data.mdx.subcomponents} componentName="sheet.header" />
+
+Nest a Header within the [Sheet](#sheet) component. If neither [Title][#sheettitle] nor
+[Description][#sheetdescription] are used, the Header should be omitted.
```

Comment:
```suggestion
Nest a Header within the [Sheet](#sheet) component. If neither [Title](#sheettitle) nor
[Description](#sheetdescription) are used, the Header should be omitted.
```
Found this:
<img width="754" alt="Screen Shot 2022-02-14 at 3 43 41 PM" src="https://user-images.githubusercontent.com/1975639/153959261-aac58894-f274-40e8-92d1-48e8c4006839.png">

Reply:
Ahh good find! Thank you.